Library/Formula/crawl.rb

```diff
@@ -0,0 +1,37 @@
+class Crawl < Formula
+  desc "Dungeon Crawl: Stone Soup"
+  homepage "http://crawl.develz.org/"
+  version "0.17.1"
+
+  url "https://github.com/crawl/crawl.git",
+    :shallow => false,
+    :revision => "942a3d2973debefdaddbf6291e4ebc890725b920"
+
+  head "https://github.com/crawl/crawl.git",
+    :shallow => false
+
+  depends_on "cmake" => :build
+
+  option "with-tiles", "Build with graphical tiles"
+
+  def install
+    if build.with? "tiles"
+      extra_args = ["NO_PKGCONFIG=y", "CONTRIB_SDL=y", "TILES=y"]
+    else
+      extra_args = []
+    end
+
+    cd "crawl-ref/source" do
+      system "make", "install",
+        "APPLE_GCC=1",
+        "prefix=#{prefix}" ,
+        "DATADIR=crawl/#{version}/",
+        "SAVEDIR='~/Library/Application\\ Support/Dungeon\\ Crawl\\ Stone\\ Soup/'",
+        *extra_args
+    end
+  end
+
+  test do
+    assert_match "Crawl version #{version}", shell_output("#{bin}/crawl -version")
```

Comment:
Any other launch would launch it interactively in the terminal, or, with tiles mode, open an SDL window. This at least ensures that the executable runs and that the version is built as expected. Is it preferred that the `test` command actually launch into interactive mode?